src/styles/blocks/stars.css

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 16px;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(../../images/star.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
All modifiers should be working. Consider using classes from the previous task for them